src/simd/distances_avx.cc

```diff
@@ -215,5 +215,28 @@ fvec_L2sqr_batch_4_avx(const float* x, const float* y0, const float* y1, const f
 }
 FAISS_PRAGMA_IMPRECISE_FUNCTION_END
 
+// trust the compiler to unroll this properly
+int32_t
```

Comment:
please confirm that it is `int32_t`, not `uint32_t`

Reply:
There seems no big difference between int32_t and uint32_t